messaging_components/services/service_dummy_broker.py

```diff
@@ -0,0 +1,128 @@
+import posixpath
+import re
+import logging
+import time
+from enum import Enum
+from iqa_common.executor import Command, Execution, ExecutorAnsible, CommandAnsible, Executor
+from iqa_common.utils.tcp_util import TcpUtil
+from messaging_abstract.component import Service, ServiceStatus
+
+
+class ServiceDummyBroker(Service):
+    """
+    Implementation of a Artemis pseudo-service to manage a Server component.
+    """
+
+    MAX_ATTEMPTS = 10
+    DELAY = 3
+
+    _logger = logging.getLogger(__name__)
+
+    def __init__(self, name: str, executor: Executor, **kwargs):
+        super().__init__(name, executor)
+        self.name = "artemis-service"
+
+        self.ansible_host = kwargs.get("ansible_host", "localhost")
+        self.service_default_port = kwargs.get("artemis_port", "61616")
+        self.service_web_port = kwargs.get("broker_web_port", "8161")
+        self.service_path = posixpath.join(kwargs.get("broker_path"), "bin", "artemis-service")
+        self.service_username = kwargs.get("broker_service_user", "jamq")
```

Comment:
@michalxo One important thing. Remember to add all the new variables to the document that describes variables that can be used with messaging components.

Reply:
These variables are temporary only. Once I will make cluster work, there will be only 1 variable `data_file_path`. So this was just for testing purposes.

Reply:
https://docs.google.com/document/d/1sCynB4DXNqNIG1cjiLFnsQMR1XZGI7w9pyfn-K-hWyk/edit# updated